src/schema/v2/edition_set.ts

```diff
@@ -86,6 +105,10 @@ export const EditionSetType = new GraphQLObjectType<any, ResolverContext>({
         return { cents, currency }
       },
     },
+    published: {
```

Comment:
Same - this isn't in the edition set JSON.

Reply:
ditto